assembly/pom.xml

```diff
@@ -165,6 +169,13 @@
     <profile>
       <id>hive</id>
       <dependencies>
+        <!-- re-declare the dependency here so hive.deps.scope would take effect -->
```

Comment:
@viirya with this, `-Phive-provided` is now taking effect on `hive-shaded` jar. If this is turned on, the Spark binary distribution will not contain the jar and users should provide the individual Hive jars instead. They are also on their own for any Guava conflicts.

Reply:
okay thanks for the update!